### ISSUE TITLE
Missing Assets on Avatar Platform

### ISSUE THREAD

**Issue by Namsemag** [SWG-Source/dsrc#114]
![screenShot0104](https://user-images.githubusercontent.com/55237094/80912665-79d56500-8d36-11ea-8a8a-29160292d294.jpg)
![screenShot0105](https://user-images.githubusercontent.com/55237094/80912669-7f32af80-8d36-11ea-9d3b-a72768d0406f.jpg)

**Comment by Namsemag** [comment]
![screenShot0106](https://user-images.githubusercontent.com/55237094/80912929-66c39480-8d38-11ea-93d4-425446d949d6.jpg)

**Comment by IrishDarkshadow** [comment]
All 4 confirmed:
![image](https://user-images.githubusercontent.com/12293802/80918014-9b484800-8d5a-11ea-97c7-b2a9b60a1bbc.png)
![image](https://user-images.githubusercontent.com/12293802/80918027-aef3ae80-8d5a-11ea-9d6f-b2f1f92d0ee2.png)
![image](https://user-images.githubusercontent.com/12293802/80918042-c7fc5f80-8d5a-11ea-9768-672bde052e8d.png)

Three of those look like particle effects, the fourth looks like a deco object. Will check server and client side buildouts to confirm what each is meant to be.

**Comment by IrishDarkshadow** [comment]
These are the expected contents for each of the three rooms in question:

techhall03
	object/static/particle/shared_pt_campfire_s01.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_poi_broken_electronics.iff
	
barracksanteroom
	object/tangible/dungeon/avatar_platform/shared_avatar_terminal_technical_readout.iff
	object/tangible/furniture/modern/shared_cabinet_modern_style_01.iff
	object/tangible/furniture/modern/shared_cabinet_modern_style_01.iff
	object/tangible/dungeon/avatar_platform/shared_avatar_terminal_technical_readout.iff
	object/tangible/furniture/modern/shared_chair_modern_style_01.iff
	object/tangible/furniture/modern/shared_chair_modern_style_01.iff
	object/tangible/furniture/modern/shared_chair_modern_style_01.iff
	object/static/structure/general/shared_data_terminal_s3.iff
	object/static/structure/general/shared_distant_ship_controller2.iff
	object/static/item/shared_item_tool_meson_taloscope.iff
	object/static/structure/corellia/shared_corl_power_box_node.iff
	object/static/structure/corellia/shared_corl_power_box_node.iff
	object/static/structure/corellia/shared_corl_imprv_wall_4x16_s03.iff
	object/static/structure/general/shared_streetlamp_small_red_style_02_on.iff
	object/static/structure/corellia/shared_corl_imprv_wall_4x16_s03.iff
	object/static/structure/corellia/shared_corl_power_box_node.iff
	object/static/structure/corellia/shared_corl_power_box_node.iff
	object/static/item/shared_item_crate_spice.iff
	object/static/item/shared_item_comp_blaster_cannon.iff
	object/static/structure/general/shared_escape_pod.iff
	object/static/structure/general/shared_data_terminal_s4.iff
	object/static/item/shared_item_con_barrel_lg_s1.iff
	object/static/item/shared_item_con_barrel_lg_s1.iff
	object/static/structure/general/shared_data_terminal_wall_s1.iff
	object/static/structure/general/shared_data_terminal_wall_s2.iff
	object/static/item/shared_item_droid_magnet.iff
	object/static/item/shared_item_security_scanner.iff
	object/static/item/shared_item_security_scanner.iff
	object/static/structure/general/shared_data_terminal_wall_s3.iff
	object/static/item/shared_item_mockup_bank_terminal.iff
	object/static/item/shared_item_comp_blaster_cannon.iff
	object/static/structure/corellia/shared_corl_power_box_node.iff
	object/static/item/shared_item_tech_console_sectional_b.iff
	object/static/item/shared_item_tech_console_sectional_b.iff
	object/static/structure/general/shared_streetlamp_small_red_style_02_on.iff
	object/static/structure/corellia/shared_corl_imprv_column_s03.iff
	object/static/structure/corellia/shared_corl_imprv_column_s03.iff
	object/static/particle/shared_pt_sparks_large.iff
	object/static/particle/shared_pt_burning_smokeandembers_md.iff
	object/static/particle/shared_pt_steam_rise.iff
	object/static/particle/shared_pt_poi_broken_electronics.iff
	object/static/particle/shared_pt_campfire_s01.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	
commandcorridor
	object/tangible/furniture/all/shared_frn_all_toolchest_lg_s01.iff
	object/tangible/furniture/all/shared_frn_all_desk_map_table.iff
	object/tangible/furniture/all/shared_frn_all_data_terminal_wall_s1.iff
	object/tangible/furniture/all/shared_frn_all_professor_desk.iff
	object/tangible/furniture/all/shared_frn_all_professor_desk.iff
	object/tangible/furniture/all/shared_frn_all_professor_desk.iff
	object/tangible/furniture/all/shared_frn_all_professor_desk.iff
	object/tangible/furniture/all/shared_frn_all_desk_radar_topology_screen.iff
	object/tangible/furniture/all/shared_frn_all_desk_radar_topology_screen.iff
	object/tangible/furniture/all/shared_frn_all_data_terminal_wall_s1.iff
	object/tangible/furniture/all/shared_frn_all_command_console.iff
	object/building/kashyyyk/shared_frn_all_cyborg_parts_rack.iff
	object/building/kashyyyk/shared_frn_all_cyborg_parts_rack.iff
	object/tangible/droid/shared_navicomputer_6.iff
	object/tangible/droid/shared_droid_space_memory_module_base.iff
	object/tangible/droid/shared_droid_space_memory_module_1.iff
	object/building/kashyyyk/shared_eqp_monitorscreen_text_s02.iff
	object/building/kashyyyk/shared_eqp_monitorscreen_text_s02.iff
	object/static/item/shared_item_tech_console_sectional_a.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_tech_console_sectional_a.iff
	object/static/item/shared_item_tech_console_sectional_a.iff
	object/static/item/shared_item_tech_console_sectional_a.iff
	object/static/structure/corellia/shared_corl_power_connector.iff
	object/static/item/shared_item_mockup_bank_terminal.iff
	object/static/item/shared_item_comp_blaster_cannon.iff
	object/static/item/shared_item_comp_blaster_cannon.iff
	object/static/structure/general/shared_escape_pod.iff
	object/static/item/shared_item_mockup_bank_terminal.iff
	object/static/item/shared_item_mockup_bank_terminal.iff
	object/static/item/shared_item_mockup_bank_terminal.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_scrolling_screen.iff
	object/static/item/shared_item_container_plain_s04.iff
	object/static/item/shared_item_container_weapons.iff
	object/static/item/shared_item_tool_handheld_viewscreen_s1.iff
	object/static/structure/general/shared_data_terminal_wall_s3.iff
	object/static/item/shared_item_security_scanner.iff
	object/static/item/shared_item_security_scanner.iff
	object/building/kashyyyk/shared_thm_all_wookiee_corpse_s03.iff
	object/building/kashyyyk/shared_thm_all_wookiee_corpse_s02.iff
	object/building/kashyyyk/shared_thm_all_wookiee_corpse_s01.iff
	object/static/particle/shared_pt_campfire_s01.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/particle/shared_pt_sparks_small.iff
	object/static/particle/shared_pt_sparks_fountain.iff
	object/static/particle/shared_pt_sparks_fountain.iff
	object/static/particle/shared_pt_smoke_small.iff
	object/static/particle/shared_pt_campfire_s01.iff
	object/static/particle/shared_pt_miasma_of_fog_gray.iff
	object/static/item/shared_item_sclpt_stuffed_bantha.iff

**Comment by IrishDarkshadow** [comment]
**First missing appearance:**

techhall03
object/static/particle/shared_pt_campfire_s01.iff

**Comment by IrishDarkshadow** [comment]
The particle appearance linked to the techhall03 object doesn't exist:
![image](https://user-images.githubusercontent.com/12293802/80919101-1e1fd180-8d60-11ea-966c-eba412550174.png)

**Comment by IrishDarkshadow** [comment]
It's the same missing particle for all 4 locations. I cannot find a particle file with the name required in any client files. It seems we're dealing with a typo in the interior layout file. I duplicated the s01 particle and renamed it to s02. That then provides the following results:

![image](https://user-images.githubusercontent.com/12293802/80919542-6809b700-8d62-11ea-9fb7-f6caa0ea8071.png)
![image](https://user-images.githubusercontent.com/12293802/80919563-8cfe2a00-8d62-11ea-8eb2-51beb5674d23.png)
![image](https://user-images.githubusercontent.com/12293802/80919576-abfcbc00-8d62-11ea-90bf-3e69c452488d.png)

New file has been added to both the server & client repositories.
